### DISCUSSION TITLE
How to add global prefix for table name?

### DISCUSSION THREAD

**Discussion by phamgiahung1368** [mikro-orm/mikro-orm#2783] [Q&A] [answered]
Does anyone know how to add global prefix for table name or mikro orm not support this ?

**Comment by B4nan** [comment, marked as answer]
Implement custom naming strategy (extend the one you want to use) and just override the `classToTableName` method to always prefix, something like `return 'prefix_' + super.classToTableName(entityName)`)

**Reply by phamgiahung1368** [reply]
tks, it work :D
